### PR TITLE
Add some configuration info in README.org

### PR DESCRIPTION
**Pull request by it-is-wednesday**
Hey! I've added two sections that I happened to want and took way too long to figure out. Also, despite you package being on MELPA, it is quite easy to miss this fact; the readme currently only has a small ELPA logo at the top, so I made it more explicit.

Thanks for your work :)

### CODE REVIEW

**Comment by clemera** [comment]
Thank you! The package is not on MELPA it's on [GNU ELPA](https://elpa.gnu.org/packages/)

**Comment by it-is-wednesday** [comment]
Whoops. Fixed that :+1:

File: README.org [hunk 1]
```diff
@@ -19,9 +25,30 @@ For manual installation, clone the repository and call:
...
+
+#+BEGIN_SRC elisp
+(with-eval-after-load 'dired
+  (setq dgi-auto-hide-details-p nil))
+#+END_SRC
+
+** Enable automatically in every Dired buffer (if in Git repository)
+
+To enable =dired-git-info-mode= whenever you navigate to a Git repository, add the following snippet:
+#+BEGIN_SRC elisp
```
Comment (clemera) [inline README.org:47]: This is a good idea, I will add a function for that could you replace that section with `(add-hook 'dired-after-readin-hook 'dired-git-info-auto-enable)`?

**Review by clemera — CHANGES_REQUESTED** [review]

File: README.org (hunk 1 above)
Comment (it-is-wednesday) [reply README.org:47]: Did so!

**Comment by clemera** [comment]
Thank you!

**Pull request merged**